source/install/cloud-changelog.md

```diff
@@ -4,12 +4,76 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-11-10](#release-2022-11-10)
+- [Release 2022-10-27](#release-2022-10-27)
 - [Release 2022-10-20](#release-2022-10-20)
 - [Release 2022-10-13](#release-2022-10-13)
 - [Release 2022-10-06](#release-2022-10-06)
 - [Release 2022-09-15](#release-2022-09-15)
-- [Release 2022-09-08](#release-2022-09-08)
-- [Release 2022-09-01](#release-2022-09-01)
+
+## Release 2022-11-10
+
+### Improvements
+
+#### User Interface (UI)
+ - Implemented progressive image loading in the webapp.
+ - Renamed "Starter" plan to "Free" plan in product.
+ - Updated NPS plugin to version 1.3.0.
+ - Removed the user interface for the legacy "Mattermost Cloud" plan (removal effective on November 1st).
+ - When the "Custom Brand Text" is left blank with custom branding enabled, the default text is now hidden.
+
+#### Administration
+ - If an Admin encounters an invitation error “SMTP is not configured in **System Console**", a link to the **System Console** SMTP configuration is now included in the error message.
+ - Crashing jobs now sets the job status to "failed".
+ - Added the ability for Cloud customers to submit their feedback for wire transfers to buy a monthly subscription.
+ - Added a new schema migration to ensure the ``ParentId`` column is dropped from the ``Posts`` table. Depending on the table size, if the column is not dropped before, a significant spike in the database CPU usage is expected on MySQL databases. Writes to the table will be limited during the migration.  
```

Comment:
Is this something that a Cloud admin would need to know or be able to take action on?

Reply:
Can you confirm @isacikgoz?

Reply:
Uh, this is not something a cloud admin can take action on. Nice catch!